hrqb/tasks/employee_salary_history.py

```diff
@@ -0,0 +1,92 @@
+"""hrqb.tasks.employee_salary_history"""
```

Comment:
[Non-blocking] This is probably something that is on top of mind for you, @ghukill:

As the number of modules under `hrqb.tasks` grow, I think it'll be helpful to have documentation describing what each of these tasks do (maybe a data dictionary / schema of the outputs from `PandasPickleTask.get_dataframe`, updating `docs/pipelines.md` to include task names/parent classes [e.g., Load == `QuickbaseUpsertTask`).

Reply:
Good call.  Or maybe a table somewhere in the repo that links Quickbase tables with the ETL tasks that load them, for quick reference.  Or mermaid diagram.

Though the naming convention is followed that `LoadEmployeeAppointments` suggests it is the task that upserts data into `Employee Appointments`, you'd still have to look through what tasks _that_ relied on.

In the same way that `HRQBPipelineTask` has a method to output an ASCII hierarchical list, I think we could output a mermaid diagram as well.  I have a hunch it would look kind of squirrely at first... but maybe not!  Maybe it would look great, and a nice addition to the docs.

All in all, agreed.  When the main tables are accounted for in QB, and shifting to adjustments, I think that'd be a good time to revisit how to sketch the organization of things quickly for developers.